src/public/css/application.css

```diff
@@ -0,0 +1,4315 @@
+.govuk-link, a {
```

Comment:
I suggest we do not store CSS files in repository. They can be easily transpiled from SCCS files at build time.